### PR TITLE
bugfix(generate): removes bad imports when generating a class

### LINKED ISSUE #2288: @angular/core/testing has no exported member addProviders
> Please provide us with the following information:
> ---------------------------------------------------------------

1. OS?  Ubuntu 16.04 LTS

2. Versions. Please run `ng --version`. If there's nothing outputted, please run
in a Terminal: `node --version` and paste the result here:
```
angular-cli: 1.0.0-beta.15
node: 6.5.0
os: linux x64
```
3. Repro steps. Was this an app that wasn't created using the CLI? What change did you
   do on your code? etc.

I created a new `Class` using the `ng g class ` generator. In this case, it was a class called `user`, so I got a `user.ts` and `user.spec.ts`
The generated `user.spec.ts` had the following content:
```
/* tslint:disable:no-unused-variable */

import { addProviders, async, inject } from '@angular/core/testing';
import {User} from './user';

describe('User', () => {
  it('should create an instance', () => {
    expect(new User()).toBeTruthy();
  });
});
```
The `addProviders` give an error because it doesn't exist in `@angular/core/testing`

4. The log given by the failure. Normally this include a stack trace and some
   more information.
```
ERROR in [default] [...]/src/app/core/user/user.spec.ts:3:9 
Module '"[...]/node_modules/@angular/core/testing/index"' has no exported member 'addProviders'.
```

5. Mention any other details that might be useful.

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

### PR DESCRIPTION
**Pull request by deebloo**
closes #2288

### CODE REVIEW

**Comment by JohannesRudolph** [comment]
@deebloo unfortunately the build failed. can you rebase and retest your pull request?

File: tests/e2e/tests/generate/class.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+export default function() {
+  // Create the pipe in the same directory.
+  const classDir = join('src', 'app');
+
+  return ng('generate', 'class', 'test-class')
+    .then(() => expectFileToExist(classDir))
+    .then(() => expectFileToExist(join(classDir, 'test-class.ts')))
+    .then(() => expectFileToExist(join(classDir, 'test-class.spec.ts')))
```
Comment (filipesilva) [inline tests/e2e/tests/generate/class.ts]: This just changed to default to not having a spec in https://github.com/angular/angular-cli/pull/2475 I believe.

**Comment by filipesilva** [comment]
Hey, thanks for fixing this! Don't worry too much about keeping the PR updated though, I usually batch-rebase/merge them myself. 

In your case though, a couple of hours ago a change landed that removed the default specs from classes so it really needs to be rebased.

**Comment by deebloo** [comment]
@filipesilva oh boy does it!

**Comment by deebloo** [comment]
@filipesilva made a mess. Will open new PR

**Pull request closed without merging**